mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -221,7 +221,8 @@ def Rock_AttentionOp :
     Rock_GemmFeaturesAttr:$features,
     OptionalAttr<I32Attr>:$numCU,
     OptionalAttr<RockTuningParamAttrInterface>:$params0,
-    OptionalAttr<RockTuningParamAttrInterface>:$params1
+    OptionalAttr<RockTuningParamAttrInterface>:$params1,
+    I32Attr:$firstGemmIdx
```

Comment:
`IndexAttr` maybe?

Reply:
I ll sneak it in a next PR or do a follow up. I think same could be said for numCU as well.